officedocs-dev-word-pia/xml/Microsoft.Office.Interop.Word/Paragraphs.xml

```diff
@@ -57,7 +57,7 @@
       </Parameters>
       <Docs>
         <param name="Range">Optional <b>Object</b>. The range before which you want the new paragraph to be added. The new paragraph doesn't replace the range.</param>
-        <summary>Returns a <see cref="T:Microsoft.Office.Interop.Word.Paragraph" /> object that represents a new, blank paragraph added to a document.</summary>
+        <summary>Returns a <see cref="T:Microsoft.Office.Interop.Word.Paragraph" /> object, When the caller is not the last paragraph in a document that represents a new, blank paragraph added to a document, otherwise that will be the calling paragraph itself.</summary>
```

Comment:
```suggestion
        <summary>Returns a <see cref="T:Microsoft.Office.Interop.Word.Paragraph" /> object. When the caller is not the last paragraph in a document that represents a new, blank paragraph added to a document, otherwise that will be the calling paragraph itself.</summary>
```

Reply:
OK. Thank you.